packages/logger/src/index.ts

```diff
@@ -56,19 +45,23 @@ const getNamespace = (namespace: string): string => {
   return namespace;
 };
 
+const debugCondition = (namespace):Boolean => {
+  return ENV.logLevel === 'debug' && minimatch(namespace, ENV.debug);
```

Comment:
The `winston` package allows us to filter by severity, but the `debug` package does not. However, the `debug` package lets us select what debug logs to show by namespace. So I added the ability for `winston` to the same, using the same `DEBUG` env var.